app/config/FrontendAppConfig.scala

```diff
@@ -74,11 +74,13 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val enrolmentStoreProxyUrl: String =
     s"${configuration.get[Service]("microservice.services.enrolment-store-proxy").baseUrl}${configuration.get[String]("microservice.services.enrolment-store-proxy.startUrl")}"
 
-  lazy val lostUTRUrl: String             = "https://www.gov.uk/find-lost-utr-number"
-  lazy val businessTaxAccountLink: String = "https://www.gov.uk/guidance/sign-in-to-your-hmrc-business-tax-account"
-  lazy val emailEnquiries: String         = "enquiries.aeoi@hmrc.gov.uk"
-  lazy val countryCodeJson: String        = configuration.get[String]("json.countries")
-  val enrolmentKey: String => String      = (serviceName: String) => configuration.get[String](s"keys.enrolmentKey.$serviceName")
+  lazy val lostUTRUrl: String                  = "https://www.gov.uk/find-lost-utr-number"
+  lazy val businessTaxAccountLink: String      = "https://www.gov.uk/guidance/sign-in-to-your-hmrc-business-tax-account"
+  lazy val corporationTaxEnquiriesLink: String = "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/corporation-tax-enquiries"
+  lazy val selfAssessmentEnquiriesLink: String = "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/self-assessment"
+  lazy val emailEnquiries: String              = "enquiries.aeoi@hmrc.gov.uk"
```

Comment:
@BriWak Can we place these hardcode values in the application.conf and read it from there.